site/cds_rdm/inspire_harvester/jobs.py

```diff
@@ -77,19 +73,19 @@ def validate_exclusive_arguments(self, data, **kwargs):
         until = data.get("until")
 
         # if `inspire_id` is provided, no other fields should be present
-        if inspire_id and any([on, since, until]):
+        if inspire_id and any([on, until]):
             raise ValidationError(
                 _(
-                    "When providing INSPIRE record ID for the search, all other args ('On', 'Since' and "
+                    "When providing INSPIRE record ID for the search, all other args ('On' and "
                     "'Until') are ignored. Please specify only inspire_id value."
                 )
             )
 
         # if `on` is provided, no other fields should be present
-        if on and any([inspire_id, since, until]):
+        if on and any([inspire_id, until]):
```

Comment:
for readability, could we rename this variable?
```suggestion
        if on_date and any([inspire_id, until]):
```